packages/react-sdk/docusaurus/docs/React/01-basics/02-introduction.mdx

```diff
@@ -47,7 +38,8 @@ The fastest way to get started with the SDK is by trying the [Video Tutorial](./
 
 We know that everyone consumes documentation differently, which is why we want to make sure everyone can reach their goals as fast as possible.
 
-- Go through it sequentially and look at the next pages (heading to [Installation](./basics/03-installation) next) to get a thorough introduction that covers the most important topics and gets you up to speed in the quickest way.
+- If you haven't yet checked out our [Video Tutorial](./basics/01-tutorial), we suggest starting there.
```

Comment:
@szuperaz this duplicates the note above

Reply:
I though it's worth mentioning it again as part of this "where to next" section

Reply:
```suggestion
- checkout the introductory [Video Tutorial](./basics/01-tutorial) first
```